docker-compose.yml

```diff
@@ -85,6 +89,54 @@ services:
     volumes:
       - ./apps/rummager/log:/app/log
 
+  rummager-worker-1:
+    << : *rummager
+    command: bundle exec sidekiq -C ./config/sidekiq.yml
+    depends_on:
+      - diet-error-handler
+    environment: &rummager-worker-env
+      << : *govuk-app
+      RABBITMQ_HOSTS: rabbitmq
+      RABBITMQ_VHOST: /
+      RABBITMQ_USER: guest
+      RABBITMQ_PASSWORD: guest
+      GOVUK_APP_NAME: rummager-worker-1
+      SENTRY_CURRENT_ENV: rummager-worker-1
+    ports: []
+
+  rummager-worker-2:
+    << : *rummager
+    command: bundle exec rake message_queue:listen_to_publishing_queue
```

Comment:
Are we able to call the procfile for the various commands?

Reply:
Rummager doesn't have foreman as a dependency so to execute these via the Procfile we'd need to add that and wait for the deployment.